ArduCopter/mode_autotune.cpp

```diff
@@ -84,7 +84,7 @@ float AutoTune::get_pilot_desired_climb_rate_cms(void) const
     float target_climb_rate = copter.get_pilot_desired_climb_rate(copter.channel_throttle->get_control_in());
 
     // get avoidance adjusted climb rate
-    target_climb_rate = copter.get_avoidance_adjusted_climbrate(target_climb_rate);
+    target_climb_rate = copter.mode_autotune.get_avoidance_adjusted_climbrate(target_climb_rate);
```

Comment:
we can probably drop the copter.mode_autotune qualifier here can't we?

Reply:
We're actually in that funny "AutoTune" object which was created to allow use of AutoTune (the library) for both Copter and Plane.  If you look at its other methods you will note that this "AutoTune" object is already calling several methods on copter.mode_autotune, so I'm following the pattern there.